### DISCUSSION TITLE
Raw html link tags appear in comments

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#53963] [Mobile]
GitHub Mobile beta

https://github.com/community/community/discussions/18455#discussioncomment-3847206
![Screenshot_20230428-091519.png](https://user-images.githubusercontent.com/2119212/235157651-d4153104-3cc2-4759-b4e4-d98618bcdcda.png)

This isn't readable, and I'd like to be able to read the content without having to switch to a browser.

**Comment by taki-on** [comment]
We render HTML in the comments, but it looks like the comment you posted above has the HTML source code as its comment body. If the author has copied and pasted an HTML source, then we render the source code as it is. If the body was a markdown or a pure text but if you see it as its HTML source, then this is a bug. Do you know what was the source content in the comment? Thanks!

**Reply by jsoref** [reply]
I don't, but I ran into this more than once in a quick span.

**Reply by jsoref** [reply]
If I were you, I'd reach out to the users and ask, or check the logs to see what they did. I'm certainly hoping you'll do that.

Thanks
